src/reactIntl.js

```diff
@@ -0,0 +1,10 @@
+export const intl = {
+	formatMessage: (trn) => trn.defaultMessage || 'formatted message',
+	formatDate: (date) => date || 'formatted date',
+	formatTime: (time) => time || 'formatted time',
+	formatRelative: (time) => time || 'formatted relative time',
+	formatNumber: (num) => num || 'formatted number',
+	formatPlural: (str) => str || 'formatted plural string',
+	formatHTMLMessage: (trn) => trn.defaultMessage || 'formatted message',
```

Comment:
might want to make the fallback string specific that this is `formatted HTML message`, just in case a test was expecting `formatMessage` to have been used